test/fuzz/basic.ml

```diff
@@ -514,6 +514,56 @@ let data_drop (env : Env.t) =
     (fun name -> pair (const (Data_drop (Symbolic name))) (const [ S.Nothing ]))
     env.datas
 
+let elem_drop (env : Env.t) =
+  List.map
+    (fun (name, _) -> pair (const (Elem_drop (Symbolic name))) (const [ S.Nothing ]))
+    env.elems
+
+let table_init (env : Env.t) =
+  List.flatten @@
+  List.map
+    (fun (name_x, _) -> 
+      List.map (fun (name_y, _) -> pair (const (Table_init (Symbolic name_x, Symbolic name_y) )) (const [ S.Pop; S.Pop; S.Pop ]))
+      env.tables
+    )
+    env.tables
```

Comment:
I'm not sure I understand what you're doing here. :-)

The `table.init` instruction expects a table index and an element index.

For instance, in `table.init $t_i $e_i`, `$t_i` must be a table and `$e_i` must be an element.

Here your are using two table indices !